service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/StoragetransferConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.workspace.service.resource.controlled.flight.clone.bucket;
+
+import com.google.api.services.storagetransfer.v1.Storagetransfer;
+import java.io.IOException;
+import org.springframework.beans.factory.BeanCreationException;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class StoragetransferConfiguration {
+
+  @Bean
+  public Storagetransfer getStoragetransfer() {
+    try {
+      return StorageTransferServiceUtils.createStorageTransferService();
```

Comment:
Looking at the test failures: `StorageTransferServiceUtils.createStorageTransferService()` tries to read Application Default Credentials, but those aren't (and shouldn't be) provided for unit tests. Putting this in a Bean this way means that startup is now dependent on having ADCs available, which breaks unit tests.

In other places we need ADCs, we've either gated the call behind a config option that we disable for unit tests ([like this for Cloud Resource Library](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/service/crl/CrlService.java#L78)), or caught the exception -> logged a warning saying "this is fine for unit tests but not anything else" -> mock out the object that needs ADC whenever it's actually used ([like this for SamService](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/service/iam/SamService.java#L158))